### PR TITLE
[ADD] gap_analysis_quotation

### PR DESCRIPTION
**Pull request by xie8899**

### CODE REVIEW

File: gap_analysis_quotation/README.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+{gap_analysis_quotation}
+==============
+
+having cost estimation and a new wizard to generate proposal/quotation based on the cost/price estimation on the business requirements.
```
Comment (elicoidal) [inline gap_analysis_quotation/README.rst]: This module is part of a set ("Gap Analysis") and extends the functionality
adding the possibility to generate/update a sales quotation based on the 
cost/price estimation of the business requirements.

File: gap_analysis_quotation/README.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Installation
+============
+
+To install this module, you need to:
+
```
Comment (elicoidal) [inline gap_analysis_quotation/README.rst]: No specific steps required for installation

File: gap_analysis_quotation/README.rst [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
```
Comment (elicoidal) [inline gap_analysis_quotation/README.rst]: No specific steps required for configuration

File: gap_analysis_quotation/README.rst [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+Usage
+=====
+
+To use this module, you need to:
+
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/{repo_id}/{branch}
```
Comment (elicoidal) [inline gap_analysis_quotation/README.rst]: modify target accordingly

File: gap_analysis_quotation/README.rst [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+* ...
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/
+{project_repo}/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
+{project_repo}/issues/new?body=module:%20
```
Comment (elicoidal) [inline gap_analysis_quotation/README.rst]: modify accordingly

**Comment by rafaelbn** [comment]
I add WIP label till readme is improved and runbot and travis are fixed

**Comment by rafaelbn** [comment]
This PR depends in https://github.com/OCA/project/pull/127

**Comment by elicoidal** [comment]
Hi @rafaelbn 
Thanks for the review and additional links.
How to fix the runbot dependencies part?

**Comment by elicoidal** [comment]
@rafaelbn Actually same question for Travis AFAICS

**Pull request closed without merging**